webapp/views/Guest/Guest.js

```diff
@@ -0,0 +1,39 @@
+import './Guest.scss'
+import React from 'react'
+import { Route, Switch } from 'react-router'
+
+import { guestModules } from '@webapp/app/appModules'
+import ResetPassword from '@webapp/views/Guest/views/ResetPassword'
+import ForgotPassword from '@webapp/views/Guest/views/ForgotPassword'
+import Login from '@webapp/views/Guest/views/Login'
+
+const WordSplitter = ({ word }) => word.split('').map((letter, i) => <div key={String(i)}>{letter}</div>)
```

Comment:
If the key has to be unique, this will not work?
Ex. same page has multiple `WordSplitter`s => same key multiple times (`0..n`)

Reply:
nope. the key is String(i) not the letter

Reply:
But what is i ?
0...word.length each time, right?

Reply:
Even if it's the letter it will not work?
ex.
`bar`
`baz`

=> both have key `a`?